resources/assets/app.scss

```diff
@@ -2,3 +2,26 @@ $forge-path: "~@dosomething/forge/assets/";
 
 // Import Forge.
 @import "~@dosomething/forge/scss/forge";
+
+.tag {
+  list-style-type: none;
+  clear: both;
+  background: $light-gray;
+  color: $med-gray;
+  font-size: $font-regular;
+  font-weight: $weight-normal;
+  font-family: $font-proxima-nova;
+  text-align: center;
+  border: 0;
+  margin: 10px 10px 0 0;
+
+  &:hover {
+    background: $blue;
+    color: white;
+  }
+
+  &:active {
+    background: $blue;
+    color: white;
+  }
```

Comment:
@DFurnes this isn't working as expected :-/ When clicked, I want it to have a blue background with white font, similar to when it's hovered. Instead, it only has a blue border. See below (left is active and right is hover state). I want them to be the same! Any ideas? 

Also, it seems like only one button is able to be active at a time (when I click on another button, the border then highlights that one). Is this something we fix with React? 

![screen shot 2017-03-31 at 1 01 08 pm](https://cloud.githubusercontent.com/assets/9019452/24560857/69ccb918-1612-11e7-9074-7b1fc56060c2.png)